### ISSUE TITLE
[@stacks/cli] `make_keychain` broken, node 18

### ISSUE THREAD

**Issue by kyranjamie** [stx-labs/stacks.js#1449]
```
/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/ajv/dist/compile/index.js:120
        throw e;
        ^

Error: strict mode: unknown keyword: "help"
    at checkStrictMode (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/ajv/dist/compile/util.js:174:15)
    at checkUnknownRules (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/ajv/dist/compile/util.js:32:13)
    at checkKeywords (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/ajv/dist/compile/validate/index.js:120:34)
    at validateFunctionCode (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/ajv/dist/compile/validate/index.js:19:9)
    at Ajv.compileSchema (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/ajv/dist/compile/index.js:80:45)
    at Ajv._compileSchemaEnv (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/ajv/dist/core.js:472:37)
    at Ajv.compile (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/ajv/dist/core.js:159:38)
    at Ajv.validate (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/ajv/dist/core.js:150:22)
    at checkArgs (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/@stacks/cli/dist/argparse.js:3137:23)
    at CLIMain (/Users/kyranjamie/.npm/_npx/e6a5a2029e8e7230/node_modules/@stacks/cli/dist/cli.js:1276:46)

Node.js v18.13.0
```

**Comment by techno369** [comment]
Same problem here with node v14.21.3 and v18.14.0. Using nvm. The same error with any commands like stx docs, stx help, etc.

**Comment by janniks** [comment]
Thanks for the reports, I will look into this

**Comment by will-corcoran** [comment]
Please note executing this command is a step early on in the 100 Days of Clarity tutorial series.  Any idea when this will be fixed? 

EDIT: on node version v19.7.0

cc: @setzeus

**Comment by janniks** [comment]
I think I found the issue, releasing a fix soon

Fixed in version `6.2.1`
